todoapp/todos/utils.py

```
@@ -15,13 +26,17 @@
 # Note: use serializer for generating this format.
 # use json.load(json.dumps(serializer.data)) while returning data from this function for test cases to pass.
 
+
 def fetch_all_users():
```

Ruff (0.12.2)

Missing return type annotation for public function `fetch_all_users`

(ANN201)